pkg/constants/constants.go

```diff
@@ -84,5 +84,10 @@ const (
 	// #nosec G101
 	GithubAPITokenEnvVarName = "GITHUB_TOKEN"
 
-	ReposDir = "repos"
+	ReposDir  = "repos"
+	SubnetDir = "subnets"
+	VMDir     = "vms"
+
+	GitRepoCommitName  = "Avalanche-CLI"
+	GitRepoCommitEmail = "info@avax.network"
```

Comment:
Is this a real email? I'm not sure we want to override the global user and email. Especially when it comes to commit signing and stuff like that.

Reply:
OK so this library will clone a repo locally. When it wants to publish, it first basically does a git-add and a git-commit locally before a git-push to the (given) remote.

Any git commit action requires a name and an email for the repo config.

I don't think this library overrides the global user and email, nor do I think that it can easily get the global user and email. I guess what it can do is try executing `git config` via a os.Exec and get it from there, but that does not guarantee we get a value (scenario: new machine setup on EC2, git just installed but not configured yet).

I guess what we could do is try the above, then if we get a value, use that, otherwise use a generic config as above.

The other approach would be to leave as-is, then in whatever repo it gets pushed, it clearly can be identified as pushed by the tool (CLI). This could also be interesting to be able to discern.

Reply:
I'm not really satisfied with this approach, though I don't immediately know the answer. I think there are some corner cases that might cause problems and I'm not sure it will meet the security level we want to provide. I think this is ok for beta version 1, but let's spend some time next week and come up with a really concrete plan.

Reply:
I think you good a job, the design spec was just unclear. This part's on me.